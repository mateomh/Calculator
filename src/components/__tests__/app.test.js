import React from 'react';
import renderer from 'react-test-renderer';
import App from '../app';

it('Quotes component renders correctly', () => {
  const tree = renderer
    .create(<App />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
