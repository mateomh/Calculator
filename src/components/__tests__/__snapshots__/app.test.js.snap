// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quotes component renders correctly 1`] = `
Array [
  <div
    className="display"
  >
    0
  </div>,
  <div
    className="buttonRow"
  >
    <button
      className="button type3"
      onClick={[Function]}
      type="button"
      value="AC"
    >
      AC
    </button>
    <button
      className="button type3"
      onClick={[Function]}
      type="button"
      value="+/-"
    >
      +/-
    </button>
    <button
      className="button type3"
      onClick={[Function]}
      type="button"
      value="%"
    >
      %
    </button>
    <button
      className="button type2"
      onClick={[Function]}
      type="button"
      value="/"
    >
      /
    </button>
  </div>,
  <div
    className="buttonRow"
  >
    <button
      className="button type1"
      onClick={[Function]}
      type="button"
      value="7"
    >
      7
    </button>
    <button
      className="button type1"
      onClick={[Function]}
      type="button"
      value="8"
    >
      8
    </button>
    <button
      className="button type1"
      onClick={[Function]}
      type="button"
      value="9"
    >
      9
    </button>
    <button
      className="button type2"
      onClick={[Function]}
      type="button"
      value="X"
    >
      X
    </button>
  </div>,
  <div
    className="buttonRow"
  >
    <button
      className="button type1"
      onClick={[Function]}
      type="button"
      value="4"
    >
      4
    </button>
    <button
      className="button type1"
      onClick={[Function]}
      type="button"
      value="5"
    >
      5
    </button>
    <button
      className="button type1"
      onClick={[Function]}
      type="button"
      value="6"
    >
      6
    </button>
    <button
      className="button type2"
      onClick={[Function]}
      type="button"
      value="-"
    >
      -
    </button>
  </div>,
  <div
    className="buttonRow"
  >
    <button
      className="button type1"
      onClick={[Function]}
      type="button"
      value="1"
    >
      1
    </button>
    <button
      className="button type1"
      onClick={[Function]}
      type="button"
      value="2"
    >
      2
    </button>
    <button
      className="button type1"
      onClick={[Function]}
      type="button"
      value="3"
    >
      3
    </button>
    <button
      className="button type2"
      onClick={[Function]}
      type="button"
      value="+"
    >
      +
    </button>
  </div>,
  <div
    className="buttonRow lastRow"
  >
    <button
      className="button type1"
      onClick={[Function]}
      type="button"
      value="0"
    >
      0
    </button>
    <button
      className="button type1"
      onClick={[Function]}
      type="button"
      value="."
    >
      .
    </button>
    <button
      className="button type2"
      onClick={[Function]}
      type="button"
      value="="
    >
      =
    </button>
  </div>,
]
`;
